src/ert/_c_wrappers/enkf/enkf_state.py

```diff
@@ -35,7 +35,7 @@ def _internalize_GEN_DATA(
             with open(run_path / filename, "r") as f:
                 data = [float(v.strip()) for v in f.readlines()]
 
-            run_arg.sim_fs.save_gen_data(f"{key}-{i}", data, run_arg.iens)
+            run_arg.sim_fs.save_gen_data(f"{key}@{i}", data, run_arg.iens)
```

Comment:
Why are we changing the format here?

Reply:
@ is the original delimiter used in ERT. Not sure if that can cause a problem later